task/net/src/mgmt.rs

```diff
@@ -219,15 +222,19 @@ impl Bsp {
         let mut s = ManagementLinkStatus::default();
         let rw = &mut MiimBridge::new(eth);
 
-        for i in 0..2 {
+        for (i, port) in
+            [KszPhyPort::One, KszPhyPort::Two].into_iter().enumerate()
+        {
             // The KSZ8463 numbers its ports starting at 1 (e.g. P1MBSR)
-            let port = i as u8 + 1;
             match self.ksz8463.read(KszRegister::PxMBSR(port)) {
                 Ok(sr) => {
                     s.ksz8463_100base_fx_link_up[i] = (sr & (1 << 2)) != 0
```

Comment:
If I'm not mistaken, the need to enumerate() this iterator could be avoided if this link_up array were an enum map keyed by KszPort.

Reply:
I looked at this, but it gets messy – Humility uses this type for the `net status` command, so changing it would require teaching Humility how to decode an `EnumMap`.